packages/plugin-ext/src/main/browser/test-main.ts

```diff
@@ -624,4 +625,8 @@ export class TestingMainImpl implements TestingMain {
     $notifyTestRunEnded(controllerId: string, runId: string): void {
         this.withController(controllerId).withRun(runId).ended();
     }
+
+    $notifyOnTestRunRequest(preserveFocus: boolean): void {
```

Comment:
Can't we use `$notifyTestRunCreated` instead of adding a new notification? A test run starts either at creation or when it's "running" flage changes, right?

Reply:
done on new commit.